Amazon/Marketplace/v7153/templates/jfrog-artifactory-core-infrastructure.template.yaml

```diff
@@ -0,0 +1,408 @@
+    m5d.xlarge:
+      Min: 8
+      Max: 12
+      DeploymentSize: xSmall
+    m5d.2xlarge:
+      Min: 16
+      Max: 24
+      DeploymentSize: Small
+    m5d.4xlarge:
+      Min: 32
+      Max: 48
+      DeploymentSize: Medium
+    m5d.8xlarge:
+      Min: 64
+      Max: 96
+      DeploymentSize: Large
+    m5d.12xlarge:
+      Min: 96
+      Max: 144
+      DeploymentSize: xLarge
+    m5d.16xlarge:
+      Min: 128
+      Max: 192
+      DeploymentSize: xxLarge
+    m5d.24xlarge:
+      Min: 192
+      Max: 288
+      DeploymentSize: xxxLarge
+    m5d.metal:
+      Min: 192
+      Max: 288
+      DeploymentSize: xxxLarge
+    m5a.large:
+      Min: 4
+      Max: 4
+      DeploymentSize: xxSmall
+    m5a.xlarge:
+      Min: 8
+      Max: 12
+      DeploymentSize: xSmall
+    m5a.2xlarge:
+      Min: 16
+      Max: 24
+      DeploymentSize: Small
+    m5a.4xlarge:
+      Min: 32
+      Max: 48
+      DeploymentSize: Medium
+    m5a.8xlarge:
+      Min: 64
+      Max: 96
+      DeploymentSize: Large
+    m5a.12xlarge:
+      Min: 96
+      Max: 144
+      DeploymentSize: xLarge
+    m5a.16xlarge:
+      Min: 128
+      Max: 192
+      DeploymentSize: xxLarge
+    m5a.24xlarge:
+      Min: 192
+      Max: 288
+      DeploymentSize: xxxLarge
+    m5ad.large:
+      Min: 4
+      Max: 4
+      DeploymentSize: xxSmall
+    m5ad.xlarge:
+      Min: 8
+      Max: 12
+      DeploymentSize: xSmall
+    m5ad.2xlarge:
+      Min: 16
+      Max: 24
+      DeploymentSize: Small
+    m5ad.4xlarge:
+      Min: 32
+      Max: 48
+      DeploymentSize: Medium
+    m5ad.12xlarge:
+      Min: 96
+      Max: 144
+      DeploymentSize: xLarge
+    m5ad.24xlarge:
+      Min: 192
+      Max: 288
+      DeploymentSize: xxxLarge
+
+Resources:
+  ArtifactoryDatabaseSubnetGroup:
+    Type: AWS::RDS::DBSubnetGroup
+    Properties:
+      DBSubnetGroupDescription: Private Subnets available to the RDS Instance(s)
+      SubnetIds: !Ref SubnetIds
+  ArtifactoryDatabase:
+    Type: AWS::RDS::DBInstance
+    Properties:
+      AllocatedStorage: !Ref DatabaseAllocatedStorage
+      MultiAZ: !Ref MultiAzDatabase
+      Engine: !Ref DatabaseEngine
+      EngineVersion: !FindInMap
+        - DatabaseMap
+        - !Ref DatabaseEngine
+        - DatabaseVersion
+      MasterUsername: !Ref DatabaseUser
+      MasterUserPassword: !Ref DatabasePassword
+      DBInstanceClass: !Ref DatabaseInstance
+      DBName: !Ref DatabaseName
+      DBSubnetGroupName: !Ref  ArtifactoryDatabaseSubnetGroup
+      StorageEncrypted: true
+      VPCSecurityGroups:
+        - !Ref ArtifactoryDatabaseSG
+  ArtifactoryDatabaseSG:
+    Type: AWS::EC2::SecurityGroup
+    Properties:
+      Tags:
+        - Key: Name
+          Value: artifactory-rds-sg
+      GroupDescription: SG for RDS Instance to allow communication from the Bastion and Artifactory servers.
+      VpcId: !Ref VpcId
+      SecurityGroupIngress:
+        - IpProtocol: tcp
+          FromPort: 22
+          ToPort: 22
```

Comment:
ssh in is discouraged - you can make use of the aws ssm client that way it uses IAM for auth